rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -356,7 +358,7 @@ namespace vk
 			rsx_log.notice("GPU/driver lacks support for float16 data types. All float16_t arithmetic will be emulated with float32_t.");
 		}
 
-		CHECK_RESULT(vkCreateDevice(*pgpu, &device, nullptr, &dev));
+		CHECK_RESULT_EXT(vkCreateDevice(*pgpu, &device, nullptr, &dev), message_on_error);
```

Comment:
CHECK_RESULT_EX sounds better and is consistent with win32 extended API naming which is more familiar to people like me.

Reply:
fixed